src/main/java/net/bramp/ffmpeg/FFmpeg.java

```diff
@@ -109,19 +111,20 @@ public FFmpeg(@Nonnull String path, @Nonnull ProcessFunction runFunction) throws
   }
 
   private static BufferedReader wrapInReader(Process p) {
-    return new BufferedReader(new InputStreamReader(p.getInputStream(), Charsets.UTF_8));
+      return new BufferedReader(new InputStreamReader(p.getInputStream(), StandardCharsets.UTF_8));
   }
 
   public synchronized @Nonnull String version() throws IOException {
     if (this.version == null) {
       Process p = runFunc.run(ImmutableList.of(path, "-version"));
+      InputStream stream = p.getInputStream();
       try {
-        BufferedReader r = wrapInReader(p);
-        this.version = r.readLine();
-        IOUtils.copy(r, new NullOutputStream(), Charsets.UTF_8); // Throw away rest of the output
```

Comment:
I vaguely recall adding this to ensure, because if the stream wasn't fully read there was some edge case where the process wouldn't die.